src/shared/components/main.tsx

```diff
@@ -429,7 +429,7 @@ const MoreFeaturesBlock = () => (
 const MoreFeaturesCard = ({ icons, text }) => (
   <div className="card card-bordered w-auto bg-neutral-800 shadow-xl">
     <div className="card-body">
-      <div className="btn btn-sm btn-secondary w-fit mb-2 pointer-events-none">
+      <div className="btn btn-sm btn-secondary w-fit mb-2 pointer-events-none text-black">
```

Comment:
The text for these are black in the existing version of join-lemmy.org, but was made whte by default after the change. I set it back to using black because the white didn't contrast very well.